hotshot-task-impls/src/quorum_proposal_recv/mod.rs

```diff
@@ -146,7 +146,10 @@ impl<TYPES: NodeType, I: NodeImplementation<TYPES>, V: Versions>
                 if self.consensus.read().await.cur_view() > proposal.data.view_number()
                     || self.cur_view > proposal.data.view_number()
                 {
-                    tracing::error!("Throwing away old proposal");
+                    tracing::error!(
```

Comment:
minor, feel like this should be

```suggestion
                    tracing::warn!(
```